packages/zen-bull-netting/src/ZenBullNetting.sol

```diff
@@ -597,23 +595,25 @@ contract ZenBullNetting is Ownable, EIP712, FlashSwap {
             oracle, ethSqueethPool, oSqth, weth, zenBull, ethIntoCrab, auctionTwapPeriod
         );
 
+        emit DepositAuction(
```

Comment:
L583 posessive apostrophe typo ->     " * @dev takes in orders from MMs to buy oSQTH"